File: SWIFT/Cap05/c05ex01.swift
// c05ex01.swift

import Foundation

var I = 1
while (I <= 5) {
    print("Entre um valor numerico: ", terminator: "")
    if let inputN = readLine(), let N = Int(inputN) {
        let R = N * 3
        print("Resultado = \(R)\n")
    } else {
        print("\nEntrada invalida.\n")
        continue       
    }
    I += 1
}

print("\nTecle <Enter> para encerrar... ", terminator: "")
_ = readLine()

File: VB_NET/Cap05/c05ex01.vb
' c05ex01.vb

Imports System
Imports System.Console

Module Program
    Sub Main(args As String())
        Dim I As Integer
        Dim N As Integer
        Dim R As Integer

        I = 1
        Do While (I <= 5)
            Write("Entre um valor numerico: ")
            N = Integer.Parse(ReadLine())

            R = N * 3

            WriteLine("Resultado = " & R & vbCrLf)

            I += 1
        Loop

        Write(vbCrLf & "Tecle <Enter> para encerrar... ")
        ReadLine()
    End Sub
End Module
